arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/SstoreSection.java

```diff
@@ -131,8 +133,7 @@ private StorageFragment doingSstore(Hub hub) {
   @Override
   public void resolveUponRollback(Hub hub, MessageFrame messageFrame, CallFrame callFrame) {
     final DomSubStampsSubFragment undoingDomSubStamps =
-        DomSubStampsSubFragment.revertWithCurrentDomSubStamps(
-            this.hubStamp(), hub.callStack().currentCallFrame().revertStamp(), 0);
+        DomSubStampsSubFragment.revertWithCurrentDomSubStamps(hubStamp, callFrame.revertStamp(), 0);
```

Comment:
Fixed two issues for `SSTORE`:
- correct `HUB_STAMP`
- using the right call frame to extract the `REVERT_STAMP`